platform/ui/src/components/studyBrowser/ThumbnailEntry.js

```diff
@@ -99,13 +99,13 @@ class ThumbnailEntry extends Component {
 
   onClick = () => {
     if (this.props.onClick) {
-      this.props.onClick();
+      this.props.onClick(this.props);
```

Comment:
I'm not sure I follow why we need to pass all of `props` instead of just the click event?

Reply:
I've changed taht to pass the basic info about the item, as we need its displaySetUID to perform the viewport update.